tapgarden/interface.go

```diff
@@ -44,7 +44,7 @@ type Planter interface {
 
 	// FinalizeBatch signals that the asset minter should finalize
 	// the current batch, if one exists.
-	FinalizeBatch() (*btcec.PublicKey, error)
+	FinalizeBatch() (*MintingBatch, error)
```

Comment:
This breaks a unit test helper here: https://github.com/lightninglabs/taproot-assets/blob/c8bed246d2f3db1fd9c12cc85d0dcada18b65d59/tapgarden/planter_test.go#L235

Reply:
Yes, I found another issue with the tests, which is now fixed.